lib/containers/common.pm

```diff
@@ -76,7 +77,7 @@ sub install_docker_when_needed {
                 assert_script_run "apt-get -y install docker-ce", timeout => 260;
             }
             else {
-                if (is_sle() && script_run("SUSEConnect --status-text | grep Containers") != 0) {
+                if (is_sle() && can_build_sle_base() && script_run("SUSEConnect --status-text | grep Containers") != 0) {
```

Comment:
The `can_build_sle_base()` is not needed here.

Reply:
same here